.github/workflows/docsite-build-deploy.yml

```diff
@@ -9,7 +9,7 @@ permissions:
 jobs:
   build-and-deploy:
     concurrency: ci-${{ github.ref }}
-    runs-on: ubuntu-latest
+    runs-on: single-cell-1tb-runner
```

Comment:
Naive question:  why do we need such a large instance type to generate docs?  Would `single-cell-8c64g-runner` suffice (I believe it uses the same base image, but on a cheaper/smaller instance type)

Reply:
It's actually a good question - I think historically some of the vignettes were very memory hungry, hence the 1T choice, but it's worth giving it a try with the 64G.

Reply:
I converted several to the small runner, to save $$

Reply:
Switched to `single-cell-8c64g-runner` and doing a test run right now.

Reply:
It worked! Leaving the change.